apps/researcher/src/app/[locale]/objects/[id]/map/index.tsx

```diff
@@ -0,0 +1,45 @@
+import {PostalAddress} from '@/lib/api/definitions';
+import dynamic from 'next/dynamic';
+
+// Disable server-side rendering for the map component,
+// because this component uses the window object.
+const Map = dynamic(() => import('./map'), {
+  ssr: false,
+});
+
+interface Props {
+  address?: PostalAddress;
+}
+
+export default async function MapByAddress({address}: Props) {
+  if (!address) {
+    return null;
+  }
+
+  try {
+    const addressString = [
+      address.streetAddress,
+      address.postalCode,
+      address.addressLocality,
+      address.addressCountry,
+    ]
+      .filter(n => n)
+      .join(',');
```

Comment:
You could place these lines outside the `try...`, to make clear that it doesn't need to be handled by the `catch`.